lib/MTT/Util.pm

```diff
@@ -490,4 +492,13 @@ sub term_handler{
 	Verbose("All is done, exiting.\n");
 	exit(0);
 }
+
+#--------------------------------------------------------------------------
+
+# Trim the leading and trailing whitespaces
+sub trim_spaces {
+    my ($str) = @_;
+    $str =~ s/^\s*|\s*$//g;
```

Comment:
So the \+ is approx. 600 microseconds faster than the \*. I'll update the patch.